core/templates/dev/head/domain/exploration/InteractionObjectFactorySpec.ts

```diff
@@ -68,7 +68,7 @@ describe('Interaction object factory', function() {
         missing_prerequisite_skill_id: null
       },
       training_data: ['training_data'],
-      tagged_misconception_id: 1
+      tagged_skill_misconception_id: null
```

Comment:
Instead of null, should we be testing a 'skill_id-misconception_id' value somewhere?

Reply:
Done.